app/src/androidTest/java/org/mozilla/fenix/ui/SettingsPrivacyTest.kt

```diff
@@ -705,6 +705,90 @@ class SettingsPrivacyTest {
             }
         }
     }
+
+    @SmokeTest
+    @Test
+    fun verifyCookieBannerReductionTest() {
+        val webSite = TestAssetHelper.getCookieBannerAsset(mockWebServer)
+
+        homeScreen {
+        }.openNavigationToolbar {
+        }.enterURLAndEnterToBrowser(webSite.url) {
+            waitForPageToLoad()
+            verifyCookieBannerExists(exists = true)
+        }.openThreeDotMenu {
+        }.openSettings {
+        }.openCookieBannerReductionSubMenu {
+            verifyCookieBannerView(isCookieBannerReductionChecked = false)
+            clickCookieBannerReductionToggle()
+            verifyCheckedCookieBannerReductionToggle(isCookieBannerReductionChecked = true)
+            exitMenu()
+        }
+        browserScreen {
+            verifyCookieBannerExists(exists = false)
+        }
+        //
+        // restartApp(activityTestRule)
+        //
+        // browserScreen {
+        //     verifyCookieBannerExists(exists = false)
+        // }.openThreeDotMenu {
+        // }.openSettings {
+        // }.openCookieBannerReductionSubMenu {
+        //     clickCookieBannerReductionToggle()
+        //     verifyCheckedCookieBannerReductionToggle(false)
+        //     exitMenu()
+        // }
+        // browserScreen {
+        // }.openThreeDotMenu {
+        // }.refreshPage {
+        //     verifyCookieBannerExists(exists = false)
+        // }
+    }
+
+    // @SmokeTest
+    // @Test
+    // fun verifyCookieBannerReductionInPrivateBrowsingTest() {
+    //     val webSite = "voetbal24.be"
+    //
+    //     homeScreen {
+    //     }.togglePrivateBrowsingMode()
+    //
+    //     navigationToolbar {
+    //     }.enterURLAndEnterToBrowser(webSite.toUri()) {
+    //         waitForPageToLoad()
+    //         verifyCookieBannerExists(exists = true)
+    //     }.openThreeDotMenu {
+    //     }.openSettings {
+    //     }.openCookieBannerReductionSubMenu {
+    //         verifyCookieBannerView(isCookieBannerReductionChecked = false)
+    //         clickCookieBannerReductionToggle()
+    //         verifyCheckedCookieBannerReductionToggle(isCookieBannerReductionChecked = true)
+    //         exitMenu()
+    //     }
+    //     browserScreen {
+    //         verifyCookieBannerExists(exists = false)
+    //     }
+    //
+    //     restartApp(activityTestRule)
+    //
+    //     homeScreen {
+    //     }.openTabDrawer {
+    //     }.openTab("Voetbal24") {
+    //         verifyCookieBannerExists(exists = false)
+    //     }.openThreeDotMenu {
+    //     }.openSettings {
+    //     }.openCookieBannerReductionSubMenu {
+    //         clickCookieBannerReductionToggle()
+    //         verifyCheckedCookieBannerReductionToggle(false)
+    //         exitMenu()
+    //     }
+    //     browserScreen {
+    //     }.openThreeDotMenu {
+    //     }.refreshPage {
+    //         verifyCookieBannerExists(exists = false)
+    //     }
+    // }
```

Comment:
Need some context in comment on commenting these out. Would it better to disable these instead?

Reply:
You can ignore this Aaron, I've commented out these steps for myself while trying to update the gecko pref.